xain_fl/coordinator/store.py

```diff
@@ -13,73 +13,57 @@
 
 
 class AbstractStore(abc.ABC):
-
-    """An abstract class that defines the API a store must implement.
-
-    """
+    """An abstract class that defines the API a store must implement."""
```

Comment:
if the only changes to this file are about the docstrings and don't change any functionality, then i wouldn't include this in the PR, otherwise it may cause confusion in the commit history

Reply:
since it doesn't change the content/meaning of the docstring but only the formatting, i consider this simple cleanup which shouldn't require a dedicated ticket.